spec/request/v0/facilities/va_request_spec.rb

```diff
@@ -2,9 +2,20 @@
 
 require 'rails_helper'
 
-RSpec.describe 'VA GIS Integration', type: :request, team: :facilities do
+vcr_options = {
+  cassette_name: '/lighthouse/facilities',
+  match_requests_on: %i[path query],
+  allow_playback_repeats: true,
+  record: :none
+}
+
+RSpec.describe 'VA GIS Integration', type: :request, team: :facilities, vcr: vcr_options do
   include SchemaMatchers
 
+  before do
+    Flipper.enable(:facility_locator_pull_operating_status_from_lighthouse, true)
```

Comment:
I removed this from the spec and it still passes.  I even tried `Flipper.disable(:facility_locator_pull_operating_status_from_lighthouse)` and it still passes.  Is this necessary?  Should there be a test specifically for when it's enabled/disabled?

Reply:
this is a fast hack to get it out asap. I should be verifying that the hash key has a value instead of the hash key existing.